src/main/js/dynamic/GlobalConfiguration.js

```diff
@@ -24,37 +28,40 @@ class GlobalConfiguration extends React.Component {
         this.handleDelete = this.handleDelete.bind(this);
         this.createEmptyModel = this.createEmptyModel.bind(this);
 
-        const { fields, name } = this.props.descriptor;
+        const { descriptor } = this.props;
+        const { fields, name } = descriptor;
         const fieldKeys = FieldMapping.retrieveKeys(fields);
         const fieldModel = FieldModelUtilities.createEmptyFieldModel(fieldKeys, DescriptorUtilities.CONTEXT_TYPE.GLOBAL, name);
         this.state = {
             currentConfig: fieldModel,
-            currentDescriptor: this.props.descriptor,
+            currentDescriptor: descriptor,
             currentFields: fields,
             showTest: false
         };
     }
 
     componentDidMount() {
-        const fieldModel = this.state.currentConfig;
-        this.props.getConfig(fieldModel.descriptorName);
+        const { currentConfig } = this.state;
+        const { getConfig: getConfig1 } = this.props;
```

Comment:
I'm not a fan of this name. Why is it getConfig1? If it's a duplicate item there should be a better name.